src/dotnet/commands/dotnet-vstest/VSTestForwardingApp.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Cli.Utils;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.Cli
+{
+    public class VSTestForwardingApp
+    {
+        private const string vstestExename = "vstest.console.dll";
```

Comment:
We usually name Constants using PascalCase. Also, it is weird to name something exename and point it to a dll. And lastly, will this work for full framework as well? I imagine in that case, you would have a vstest.console.exe to be called without dotnet, no?